server-toss/src/migrations/Migration20260703033909.ts

```
@@ -0,0 +1,21 @@
+export class Migration20260703033909 extends Migration {
+
+  override up(): void | Promise<void> {
+    this.addSql(`alter table \`prompts\` modify \`strokes\` mediumtext not null;`);
+
+    this.addSql(`alter table \`rankings\` modify \`strokes\` mediumtext not null;`);
+
+    this.addSql(`alter table \`drawings\` modify \`strokes\` mediumtext not null;`);
+  }
+
+  override down(): void | Promise<void> {
+    this.addSql(`alter table \`prompts\` modify \`strokes\` varchar(255) not null;`);
```

ESLint

[error] Replace ``alter·table·\`prompts\`·modify·\`strokes\`·varchar(255)·not·null;`` with `⏎······`alter·table·\`prompts\`·modify·\`strokes\`·varchar(255)·not·null;`,⏎····`

(prettier/prettier)